kit/Kit.cpp

```diff
@@ -2391,6 +2391,32 @@ void wakeCallback(void* pData)
 
 #ifndef BUILDING_TESTS
 
+namespace
+{
+#if !MOBILEAPP
+void copyCertificateDatabaseToTmp(Poco::Path jailPath)
+{
+    auto aCertificatePathString = config::getString("certificates.database_path", "");
+    if (!aCertificatePathString.empty())
+    {
+        Poco::Path aCertificatePath(aCertificatePathString);
```

Comment:
Do we want to check if this path exists and is valid, and log a warning otherwise? I think that will help users troubleshoot typos in the path and things like that easily.